helpdesk_mgmt/views/helpdesk_ticket_team_view.xml

```diff
@@ -35,6 +35,25 @@
                         <field name="user_ids" widget="many2many_tags"/>
                         <field name="company_id" groups="base.group_multi_company"/>
                     </group>
+                    <group>
+                        <group name="left">
+                            <field name="active" invisible="1"/>
+                            <label for="alias_name" string="Email Alias"/>
+                            <div class="oe_inline" name="alias_def">
+                                <field name="alias_id" class="oe_read_only oe_inline"
+                                    string="Email Alias" required="0"/>
+                                <div class="oe_edit_only oe_inline" name="edit_alias" style="display: inline;" >
+                                    <field name="alias_name" class="oe_inline"/>@<field name="alias_domain" class="oe_inline" readonly="1"/>
+                                    <button  icon="fa-arrow-right" type="action" name="%(base_setup.action_general_configuration)d" string="Configure domain name" class="btn-link" attrs="{'invisible': [('alias_domain', '!=', False)]}"/>
+                                </div>
+                            </div>
+                            <field name="alias_contact" class="oe_inline"
+                                string="Accept Emails From" />
+                            <field name="alias_user_id" string="Assign to"/>
+                        </group>
+                        <group name="right">
```

Comment:
Why this extra empty group?

Reply:
copy/pastle issue... :sob: 
https://github.com/OCA/OCB/blob/13.0/addons/sales_team/views/crm_team_views.xml#L77